scripts/template_upload_all.py

```diff
@@ -416,6 +430,8 @@ def main():
 
         logger.info("TEMPLATE_UPLOAD_ALL:------End of %r upload on: %r--------",
             kwargs['template_name'], provider_type)
+    else:
+        logger.warning('No URL match, not calling any upload modules.')
```

Comment:
It would fail completely silent if the URL given by image_url doesn't match any of the URLs from the yaml file. This catches the loop exhaustion that's indicative of this behavior.